### PR TITLE
added contribution readme and refactored readme into smaller files

### LINKED ISSUE #147: docs: update getting started guide for developers
https://docs.google.com/document/d/1w2EFiGWPOLw61G5zv8n5NuNQ7bfb6ZkFyp4OLKEdRaE/edit

### PR DESCRIPTION
**Pull request by Nacho114**
The README is looking good but it's a bit overwhelming, and it's also missing a "how to contribute section".

In this brief PR we add the following:

- Add a CONTRIBUTING.md to the source folder explaining our workflow

We refactor the following:

- Break up the big README.md, we keep a minimal README.md with instructions of how to build, run and general info, and add another ARCHITECTURE.md and FAQ.md explaining details.

Will close #147

### CODE REVIEW

**Comment by Nacho114** [comment]
@gozgun thanks!! first commit to disco 🥳

File: mobile-browser-based-version/README.md [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
+Welcome to the Disco developer guide, the guide is broken down into small chapters. To get up and running you can find the information here as well as in the [SERVER](server/README.md) chapter.
+
+If you run into any sort of trouble then hopefully you can find an answer in our [FAQ](information/FAQ.md); otherwise please create a new issue.
+
+To learn more about the development (contributing) process or our architecture you can go to the respective chapters.
+
+#### Chapters
+
+- README _(you are here)_
+- [SERVER](server/README.md)
```
Comment (martinjaggi) [inline mobile-browser-based-version/README.md]: helper server, and comes last among the 4

File: mobile-browser-based-version/information/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Disco - FAQ
+
+### Problem:  Using TensorFlow.js on Mac laptops with M1 chips
+
+### Solution:
+
+`TensorFlow.js` in version `3.13.0` currently supports for M1 Mac laptops. However, make sure you have an `arm` node executable installed (not `x86`). It can be checked using:
```
Comment (martinjaggi) [inline mobile-browser-based-version/information/FAQ.md]: or later? (no need to mention if we are newer)

**Review by martinjaggi — CHANGES_REQUESTED** [review]

File: mobile-browser-based-version/README.md (hunk 1 above)
Comment (Nacho114) [reply mobile-browser-based-version/README.md]: yeah, i thought so too, but you also need to read the server readme in order to run the whole thing, whereas the rest is more about info, so i wrote it based on that order. what do u think

**Pull request merged**